### PR TITLE
Board : display external link in table view

### PR DESCRIPTION
**Pull request by morucci**

### CODE REVIEW

File: web/src/components/Board.res [hunk 1]
```diff
@@ -177,7 +177,11 @@ module Board = {
       {
         Column.name: "To Review",
         order: mkOrder("created_at", None),
-        query: "state:open and updated_at>now-3week " ++ " and not " ++ defaultNegativeApprovals,
+        query: "state:open and updated_at>now-3week " ++
+        " and not " ++
+        defaultNegativeApprovals ++
+        " and not " ++
```
Comment (TristanCacqueray) [inline web/src/components/Board.res:183]: I think you meant:
```suggestion
        " and " ++
```

File: web/src/components/Board.res [hunk 2]
```diff
@@ -163,7 +163,7 @@ module Board = {
   type t = {title: string, columns: list<Column.t>, style: style}
 
   let defaultNegativeApprovals = "(approval:CHANGE_REQUESTED or approval:Verified-1 or approval:Workflow-1 or approval:Code-Review-1 or approval:Code-Review-2)"
-  let defaultPositiveApprovals = "(approval:APPROVED or approval:Code-Review+1 or approval:Code-Review+2)"
+  let defaultPositiveApprovals = "(approval:APPROVED or approval:Code-Review+2)"
```
Comment (TristanCacqueray) [inline web/src/components/Board.res:166]: Why remove CR+1 ? This is used for the `To Review` list, not for the `To Approve` list

**Review by TristanCacqueray — CHANGES_REQUESTED** [review]

File: web/src/components/Board.res (hunk 1 above)
Comment (morucci) [reply web/src/components/Board.res:183]: The idea is to not see those already in the "to approve" section.

File: web/src/components/Board.res (hunk 2 above)
Comment (morucci) [reply web/src/components/Board.res:166]: I only want to see changes that received "real" approval in "to approve" and not in "to review" list.

File: web/src/components/Board.res (hunk 1 above)
Comment (morucci) [reply web/src/components/Board.res:183]: But well if you don't like the change I can revert and keep this only for my dashboard.

**Review by TristanCacqueray — APPROVED** [review]

**Pull request merged**